plugins/inputs/vsphere/vsan.go

```diff
@@ -440,8 +441,10 @@ func populateCMMDSTags(tags map[string]string, entityName, uuid string, cmmds ma
 				newTags["hostname"] = host.Content.Hostname
 			}
 			newTags["devicename"] = e.Content.DevName
-			if int(e.Content.IsSsd) == 0 {
-				newTags["ssduuid"] = e.Content.SsdUUID
+			if !(strings.Contains(entityName, "-esa-")) {
```

Comment:
Could you please add a comment to the code noting that the ssduuid is invalid for esa disks? If possible, add a link to a VSphere document describing that fact...